workflows: add docs for Revive workflow (#3512)
AOSC-Dev/aosc-os-abbs: .github/workflows/README.md

# Workflow Build Packages On Request (Project Revive)

## Syntax

```
/build <Package> [<Package> ...]
```

## Description

> The `/build` command builds packages on GitHub Actions. It starts clean, up-to-date instances of buildkit container, runs `acbs-build` to build packages.

Reviewer: via GitHub Actions. *

Author: ```suggestion
The `/build` command builds packages via GitHub Actions. It starts clean, up-to-date instances of buildkit container, runs `acbs-build` to build packages.
```

> When replying the command under an issue, packages are built from `stable` branch. When replying the command under a pull request, packages are built from the pull request.

Reviewer: When replying the command under an issue, packages are built from `stable` branch. When replying the command under a pull request, packages are built from the respective origin branch of the pull request.

Author: ```suggestion
When replying the command under an issue, packages are built from `stable` branch. When replying the command under a pull request, packages are built from the source branch of the pull request.
```

## Examples

> ### Example 1: Build one package

Reviewer: Building *, same for below.

Author: ```suggestion
### Example 1: Building One Package
```

```
/build aosc-aaa
```

> ### Example 2: Build multiple packages

Author: ```suggestion
### Example 2: Building Multiple Packages
```

```
/build openssl openssl+32
```

> ### Example 3: Build a group of packages

Author: ```suggestion
### Example 3: Building A Group Of Packages
```

```
/build groups/dotnet
```

## Notes

> - Currently only GitHub-hosted runners are supported.

Reviewer: Currently, this command only supports GitHub-hosted runners.

Author: ```suggestion
- Currently, this command only supports GitHub-hosted runners.
```

>   - Standard_DS2_v2 virtual machines in Microsoft Azure isn't big enough to build every package in AOSC OS, nor does it provide architectures other than Intel 64.

Reviewer: Microsoft Azure's Standard_DS2_v2 virtual machines only support the x86_64 (`amd64`) architecture, and may not have sufficient storage to build some packages.

Author: ```suggestion
  - Microsoft Azure's Standard_DS2_v2 virtual machines only support x86-64 (`amd64`) architecture and may not have sufficient performance or storage to build some packages.
```

> ## Related Links

Reviewer: Further Reading

Author: ```suggestion
## Further Reading
```
